src/Service/AttributeReader.php

```diff
@@ -0,0 +1,188 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\AppBundle\Service;
+
+use App\Attribute\ConfirmationRoute;
+use App\Attribute\RegistrationRoute;
```

Comment:
You can not refer to a namespace not existing in your own package and not covered in your dependencies. In this case you refer to attributes defined in the (AppTemplate)[https://github.com/shopware/AppTemplate/tree/main/src/Attribute] that should be using the AppBundle, not the other way around.

It would probably fit the AppModule to define the attributes for RegistrationRoute and ConfirmationRoute itself and the AppTemplate to refer to this definition.